README.rst

```diff
@@ -74,20 +74,12 @@ Usage Example
 
 Caveat: by default the library initializes the IC with constant temperature and pressure measurements at 64Hz with 64 samples. It is not possible to change the IC's mode, temperature_oversample_count or pressure_oversample_count on-the-fly so resetting the IC and operation parameteres is required. For instance, to set the mode to continuous pressure measurement at 1Hz with 2 samples:
 
-.. code-block:: python3
-
-    dps310.reset()
-    dps310.pressure_oversample_count = adafruit_dps310.SampleCount.COUNT_2
-    dps310.pressure_rate = adafruit_dps310.Rate.RATE_1_HZ
-    dps310.mode = adafruit_dps310.Mode.CONT_PRESSURE
-    dps310.wait_pressure_ready()
```

Comment:
The code that was removed here is tied into the explanation before it. It either needs restored or the Caveat needs removed as well.

Reply:
Oh I see Gotcha, will do